docker/start.py

```diff
@@ -21,7 +21,8 @@
 }
 
 sqliteWebPath = '/home/sqlite_web/sqlite_web.py'
-enableSQLITE = os.path.isfile(sqliteWebPath) and os.environ.get('SQLITE', 'false').lower() == 'true'
+enableSQLITE = os.environ.get('SQLITE', 'false').lower() == 'true'
+enableSQLITE = os.getenv('TYPE') != 'MINIMAL'
```

Comment:
So, you basically overwrite the line before... Please merge them into one `and`-line or remove the latter.